api/src/main/java/com/thesurvey/api/dto/request/AnsweredQuestionRequestDto.java

```diff
@@ -11,6 +12,7 @@ public class AnsweredQuestionRequestDto {
 
     private UUID surveyId;
 
+    @NotBlank
```

Comment:
Same reason for https://github.com/kookmin-sw/capstone-2023-40/pull/78/commits/50685a2ec3d483ef55d2f8c2262fba21684a9f99